packages/gatsby-cli/src/reporter/index.js

```diff
@@ -71,20 +71,23 @@ const interuptActivities = () => {
   })
 }
 
-const prematureEnd = () => {
+const prematureEnd = (failure = false) => {
```

Comment:
is argument needed?

Reply:
Yup, because in `panic`, I want to be able to pass in `true`

Reply:
but all calls to `prematureEnd` are passing `true`

Reply:
Not the one in the `signalExit` handler

Reply:
This one - https://github.com/gatsbyjs/gatsby/blob/b7f123df4c1cd7b7be9378343d0f5945fe7fb9c5/packages/gatsby-cli/src/reporter/index.js#L90
? :P

Reply:
I clearly need sleep hahaha

Reply:
Wait cleaning it up